libs/platform/src/lib/components/table/table.component.html

```diff
@@ -209,6 +215,8 @@
                 </fd-popover>
             </th>
         </ng-container>
+
+        <th class="fdp-table__cell--empty"></th>
```

Comment:
just curious, what is this for?

Reply:
![image](https://user-images.githubusercontent.com/20265336/125187898-7abb2e00-e23a-11eb-9082-3dc121a59d7e.png)

If we resize columns so their summary width will be less than the table container - we need to have some space (empty column).